contracts/nft/src/execute/burn.rs

```diff
@@ -0,0 +1,23 @@
+use cosmwasm_std::{DepsMut, Env, MessageInfo, Response};
+use provwasm_std::types::provenance::metadata::v1::MsgDeleteScopeRequest;
+use uuid::Uuid;
+
+use crate::core::error::ContractError;
+use crate::util::action::{Action, ActionType};
+use crate::util::metadata_address::MetadataAddress;
+
+pub fn handle(
```

Comment:
Does tokens have to be decremented here; I noticed it's incremented when you mint? I'm a little fuzzy on how scope, session, and record fit together. How come only the scope needs to be deleted and not the record?

Reply:
should be fixed now